python/requirements.txt

```diff
@@ -13,3 +13,5 @@ pyflakes == 0.8.0
 
 # For test-webidl
 ply == 3.8
+
+-e python/tidy
```

Comment:
Cool!
I never knew we can use `-e foo/bar` in requiremnets.txt before.
It's really helpful to avoid the publish problems, and make testing easier :)